### PR TITLE
Feature added support for TOUCH and OBJECT IDLETIME

### PR DESCRIPTION
**Pull request by apoorvyadav1111**
* The changes that are included in the PR.
Added support for the `TOUCH` command as per Redis and `OBJECT IDLETIME key` as per Redis.
Added tests for both commands in tests folder.

* Design document, if any.

* Information on any implementation choices that were made.
The reason to implement  `OBJECT IDLETIME key`, standalone from other subcommands is due to no support for ref and freq for keys and object for now. However, I assumed that having `object idletime` is essential to verify the effect of `TOUCH` command. In addition to this, I have carefully structured `OBJECT` for extension for subcommands in the future.

* Evidence of sufficient testing. You MUST indicate the tests done, either manually or automated.
Added new tests

### CODE REVIEW

**Comment by apoorvyadav1111** [comment]
Hi @JyotinderSingh, I have added new command support to the Dice. Please review and share your thoughts on the implementation at your convenience.

**Comment by JyotinderSingh** [comment]
> Hi @JyotinderSingh, I have added new command support to the Dice. Please review and share your thoughts on the implementation at your convenience.

Looks like the checks and linter are failing, could you take a look?

**Comment by apoorvyadav1111** [comment]
Hi @JyotinderSingh, I have checked the linter and build issue. The build issue was due to the session_test.go file at line 107, which is recently fixed by #318. For the linter, the issue comes from eval.go line 82 `core/eval.go:82:6: ineffectual assignment to password (ineffassign)` (at least from my local lint run). I could not access much logs from the GitHub run.

I can fix the build issue by rebasing my branch at the end and after addressing any comments, before Merge. Meanwhile I can also look into the lint issue.

**Review by JyotinderSingh — COMMENTED** [review]
Thanks for the PR @apoorvyadav1111! The changes look good, will approve once rebased on latest master.

**Comment by apoorvyadav1111** [comment]
Hi @JyotinderSingh , I have addressed the comments and ran the unit and integration tests locally along with the linter. Please review the changes.

**Review by JyotinderSingh — APPROVED** [review]

**Pull request merged**
